onboarding_auth/onboardingAuthPresentation/src/main/java/com/alexafit/onboardingAuthPresentation/login/LoginScreen.kt

```diff
@@ -0,0 +1,157 @@
+package com.alexafit.onboardingAuthPresentation.login
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.ScaffoldState
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.style.TextAlign
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.alexafit.core.util.UiEvent
+import com.alexafit.coreui.LightBlue
+import com.alexafit.coreui.LocalSpacing
+import com.alexafit.coreui.components.buttons.TextActionButton
+import com.alexafit.coreui.components.textfield.TextField
+import com.alexafit.onboardingAuthPresentation.R
+import com.alexafit.onboardingAuthPresentation.login.event.LoginEvent
+
+@Composable
+fun LoginScreen(
+    scaffoldState: ScaffoldState,
+    onEventClick: (LoginEvent) -> Unit,
```

Comment:
I assume this should just be `onEvent` right? This sounds like you're clicking on an event

Reply:
I think my naming is a bit off in terms of generalizing a click event. Should I name it onEvent or onClick?

Reply:
An event like `LoginEvent` could be any type of user action. This could be typing something in a text field, swiping something away, clicking a button or similar. Therefore I'm not sure what the `Click` in `onEventClick` refers to (since you're not just clicking things)